src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -144,7 +136,6 @@ public void setGridConfig(GridConfig gridConfig) {
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes